From d61c89a8214896350982578873f109dd1a70036e Mon Sep 17 00:00:00 2001
From: Raj Rajasekaran <b10872@freescale.com>
Date: Mon, 8 Sep 2008 10:35:51 -0500
Subject: [PATCH] ENGR00088034 SCC2: Use IRAM base addr instead of offset

Passed IRAM_BASE_ADDRESS to SCC2 driver instead of the address with offset.

Signed-off-by: Raj Rajasekaran <b10872@freescale.com>
---
 drivers/mxc/security/scc2_driver.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/security/scc2_driver.c b/drivers/mxc/security/scc2_driver.c
index d7b98be..1e88e80 100644
--- a/drivers/mxc/security/scc2_driver.c
+++ b/drivers/mxc/security/scc2_driver.c
@@ -112,8 +112,8 @@ static volatile void *scc_base;
     #scc_monitor_security_failure() and processed by
     #scc_perform_callbacks() */
 static void (*scc_callbacks[SCC_CALLBACK_SIZE]) (void);
-
-uint32_t scm_ram_phys_base = SCC_IRAM_BASE_ADDR;
+/*SCC need IRAM's base address but use only the partitions allocated for it.*/
+uint32_t scm_ram_phys_base = IRAM_BASE_ADDR;
 
 void *scm_ram_base = NULL;
 
-- 
1.5.4.4

